tests/e2e/tests/pilot/testdata/networking/v1alpha3/sidecar-scope-ns1-ns2.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Sidecar
+metadata:
+  name: sidecar-scope-ns1-ns2
+spec:
+  ingress:
+  - port:
+      number: 12345
+      protocol: HTTP
+      name: inbound-http
+    defaultEndpoint: ":80"
```

Comment:
This should be 127.0.0.1:80 - normal semantic for :80 is to bind on all interfaces, which would be a huge security issue for non-iptable mode. 

We may still support 1.2.3.4:80 - for middle boxes - but I don't see any good use case for :80

Reply:
Per the spec, (and per your request earlier), the short form for 127.0.0.1:80 is :80. Ingress listeners cannot have default endpoints to anything other than 127.0.0.1 or unix domain socket. If omitted, we add 127.0.0.1 for the end user